code/modules/games/cards.dm

```diff
@@ -299,7 +299,7 @@
 /obj/item/weapon/hand
 	name = "hand of cards"
 	desc = "Some playing cards."
-	icon = 'icons/obj/playing_cards.dmi'
+	icon = 'icons/obj/playing_cards_ch.dmi' //CHOMPEDIT quickest solution to having custom chomp cards - Jack
```

Comment:
Same thing as with flooring, chompedit avoids any issues doing override, maybe at later date will do override, but prefer doing what works since casino soon